static/app/views/dashboardsV2/widget/metricWidget/index.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
```

Comment:
There's a weird intermediate state for me I noticed. To reproduce:
1. Deselect any projects from the global selection header
1. Add a new widget and select metrics widget
1. pick a project in the modal pop-up
1. it stays on event widget and I have to select metrics widget again

Reply:
yes this was a regression. I just pushed a fix...thanks 🙏